src/test/java/io/cdap/plugin/hubspot/source/etl/HubspotAPISourceETLTest.java

```diff
@@ -0,0 +1,499 @@
+                                                                      "20191111",
+                                                                      "Category",
+                                                                      null,
+                                                                      "totals",
+                                                                      null,
+                                                                      "summarize/daily");
+    List<StructuredRecord> records = getPipelineResults(sourceHubspotConfig);
+  }
+
+  @Test
+  public void testAnalyticsContentDaily() throws Exception {
+    SourceHubspotConfig sourceHubspotConfig = new SourceHubspotConfig(testName.getMethodName(),
+                                                                      null,
+                                                                      "Analytics",
+                                                                      apiKey,
+                                                                      "client",
+                                                                      "20190101",
+                                                                      "20191111",
+                                                                      "Content",
+                                                                      "standard-pages",
+                                                                      null,
+                                                                      null,
+                                                                      "daily");
+    List<StructuredRecord> records = getPipelineResults(sourceHubspotConfig);
+  }
+
+  @Test
+  public void testAnalyticsObjectMonthly() throws Exception {
+    SourceHubspotConfig sourceHubspotConfig = new SourceHubspotConfig(testName.getMethodName(),
+                                                                      null,
+                                                                      "Analytics",
+                                                                      apiKey,
+                                                                      "client",
+                                                                      "20190101",
+                                                                      "20191111",
+                                                                      "Object",
+                                                                      null,
+                                                                      null,
+                                                                      "pages",
+                                                                      "monthly");
+
+    List<StructuredRecord> records = getPipelineResults(sourceHubspotConfig);
+  }
+
+  public List<StructuredRecord> getPipelineResults(SourceHubspotConfig sourceHubspotConfig) throws Exception {
```

Comment:
In one source class getPipelineResults receives map, in other a config. We probably need to make these the same. As it's a bit weird currently.